### DISCUSSION TITLE
2024/05/20/20240520_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#22] [General]
# 2024/05/20/20240520_updatelog/

又是一年520。 虽然今年此时依旧单身，但或许还是可以讨论点爱情相关的内容。几天前，看完了马尔克斯的一部小说《霍乱时期的爱情》，有了一些想法，正好放在520这个大背景下谈一谈。

https://wz.anoms.top/2024/05/20/20240520_updatelog/

**Comment by Hazelandinstant** [comment]
我最近读到非常感动的爱情小说是《一个人的朝圣2》。我之前没有读过《一个人的朝圣1》，所以所有理解都是基于2的背景。大意就是临终前一位老人（1）收到一份来信，是她年轻时候喜欢但爱而不得的人送来的，要来临终关怀养老院来看她，并且是千里迢迢步行过来的，且也是一位老人（2）。整个小说一边描写年轻时候的回忆和心动（1），一边写着这位老人（2）在路上千里迢迢跋涉过来得到很多人的帮助并且被电视台报道。最后轮椅上的老人（1）在激动地得知爱人（2）到来的消息时死去。但事实上，以上全部只是老人（1）的幻想。事实是，这位老人（1）在得知自己时日无多时，在一页页草稿纸上留下谁都认不出来的字迹，最后在寂寞中死去，所有的一切都是她的想象。而那位爱而不得的恋人，自始至终都不知道她曾经喜欢过他。

**Reply by cyclinbox** [reply]
听起来，《一个人的朝圣2》是一个爱而不得的伤心故事，主人公虽然有美好的回忆，结局却也颇为遗憾。

我没看过《朝圣2》但是看过《朝圣1》（高中时最好的朋友借给我的，很怀念那个时候），那也是一个人因为一封信走上朝圣之路的故事。但在《朝圣1》的平行宇宙中，男主最终真的凭借毅力走到了他昔日好友的临终病床前。在朝圣之路上，男主也回忆和反思了种种过往，包括年轻时的经历、和妻儿在一起时的家庭生活、儿子抑郁自杀的巨大打击以及从此开始与妻子的隔阂。最后的最后，他和自己和解、和妻子互相和解，甚至最后他和妻子一起送走了昔日好友的最后一程。从那本书中没有看出太多男主和写信人（应该是第二部的女主吧）之间的爱情，也许男主本身在情感方面就有些迟钝，也许他对待老朋友的情感中带着许多克制。男主的性格其实也带有一些缺陷，不敢肩负属于自己的责任，但是最后他反思了自己，两位老友也见上了一面，故事的最后大家也彼此和解，读到最后能感觉到人与人之间的那种温情。

另外，从你提到的《朝圣2》的剧情中，我想到了另一部很感人的作品，是一个叫做To The Moon的剧情游戏，讲述的是两位孤独症患者彼此救赎的故事，而在最后，临终的男主在做梦机器的帮助下实现了对女主的承诺，和她终于在月球上相见。这部游戏的画面和音乐也很精美，如果你感兴趣的话，可以找一找资源（网上有很多）

**Reply by Hazelandinstant** [reply]
我感觉第二部的女主就是第一部中男主的昔日好友！
在第二部中，这个昔日好友变成了爱而不得的女主，在各种细节中爱着男主，并涉入他的家庭（这也是第一部中为什么他和他的妻子一起送走昔日好友的原因吧）。这两部也算是相互呼应了。
但是本人更加偏爱第二部，因为如你所说，其实男主并不完美，只有想象以及从未接近，才能保持住美好的回忆和人设。
所以我也在想，走近每个人后，都能看到他/她的阴暗面，既然如此，还有必须走近吗？保持距离难道不够完美吗？（或许走近之后除了阴暗面也能看到更加好的一面）
也谢谢推荐的游戏！有时间一定会去看看！

**Reply by cyclinbox** [reply]
很喜欢这句话，“只有想象以及从未接近，才能保持住美好的回忆和人设”。不过，即使走近每个人后都能看到ta的阴暗面，但敢于面对这种不完美，不也是一种勇敢的表现吗？所谓“要爱具体的人，而不是爱抽象的人”，也许做一个大胆爱的人更真实也更勇敢。

感觉作者的情节安排也很有特色，第一部中作者着墨于朝圣者男主，女主的经历和心理活动却被忽略了，而这些都在第二部中得到补全，也更让读者感到遗憾和爱而不得的痛苦。在这些遗憾和痛苦的背后，是一个个真实的、带着残缺却努力活着的人生。总之非常感谢推荐这本书！有机会我也要把第二部买回来读一读！
